Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationHealthMonitoringConfigurationPolicyWindows10/MSFT_IntuneDeviceConfigurationHealthMonitoringConfigurationPolicyWindows10.psm1

```diff
@@ -0,0 +1,1382 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        #region resource generator code
+        [Parameter()]
+        [ValidateSet('notConfigured','enabled','disabled')]
+        [System.String]
+        $AllowDeviceHealthMonitoring,
+
+        [Parameter()]
+        [System.String]
+        $ConfigDeviceHealthMonitoringCustomScope,
+
+        [Parameter()]
+        [System.String]
+        $ConfigDeviceHealthMonitoringScope,
```

Comment:
Same comment as in the schema. If we change the type to an array, we can add the ValidateSet parameter, making sure only valid input is provided.